lang/ruby/lib/avro/schema_validator.rb

```diff
@@ -62,12 +70,11 @@ def to_s
     TypeMismatchError = Class.new(ValidationError)
 
     class << self
-      def validate!(expected_schema, logical_datum, options = { recursive: true, encoded: false, fail_on_extra_fields: false })
+      def validate!(expected_schema, logical_datum, options = DEFAULT_VALIDATION_OPTIONS)
         options ||= {}
```

Comment:
Should the default here be replaced with a frozen constant, or should the line be removed entirely? It would only be used if `options` is explicitly passed as something falsey.

Reply:
I think we should remove this line since I don't see the value in letting clients explicitly pass a nil `options` hash.